examples/sqlite_drizzle/src/entities/users/db.ts

```diff
@@ -0,0 +1,93 @@
+import { desc, eq } from 'drizzle-orm';
+import { integer, sqliteTable, text } from 'drizzle-orm/sqlite-core';
+
+import { DrizzleDb } from '../../db';
+
+export const Users = sqliteTable('users', {
+    id: integer('id').primaryKey(),
+    username: text('username').unique().notNull(),
+    age: integer('age').notNull()
+});
+export type User = typeof Users.$inferSelect;
+export type UserCreate = Omit<User, 'id'>;
+export type UserUpdate = Pick<User, 'id'> & Partial<UserCreate>;
+
+export async function getUsers(
+    db: DrizzleDb,
+    limit: number,
+    offset: number
+): Promise<User[]> {
+    return await db.select().from(Users).limit(limit).offset(offset);
+}
+
+export async function getUser(
+    drizzleDb: DrizzleDb,
+    id: number
+): Promise<User | null> {
+    const users = await drizzleDb.select().from(Users).where(eq(Users.id, id));
+
+    return users.length === 0 ? null : users[0];
+}
+
+export async function countUsers(drizzleDb: DrizzleDb): Promise<number> {
+    const results = await drizzleDb
```

Comment:
would it be better to use the count() method built into drizzle? (see similar comment in Post)